### ISSUE TITLE
`double-slash-comment-empty-line-before` except first-nested doesn't work

### ISSUE THREAD

**Issue by spaceninja** [kristerkari/stylelint-scss#183]
From my config:

```
        "scss/double-slash-comment-empty-line-before": [ "always", {
            except: ["first-nested"],
            ignore: ["between-comments", "stylelint-commands"],
        } ],
```

And the following code throws an error:

```css
.form-single-array {

    // nested form items
    .form-group {
        color: red;
    }
}
```

If I'm reading the rule correctly, the `except: ["first-nested"]` should cause that CSS to pass validation. Am I doing something wrong?

**Comment by kristerkari** [comment]
Thanks for reporting! I can add a unit test for that and let's see what result I get :)

**Comment by kristerkari** [comment]
@spaceninja Looking at the unit tests for that rule it's working as espected:
https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/double-slash-comment-empty-line-before/__tests__/index.js#L135-L148

So, your code reports an error with `except: ["first-nested"]` enabled, but not if that option is disabled.

**Comment by spaceninja** [comment]
🤦‍♂️ You're totally right, I misunderstood the rule. Thanks!

**Comment by kristerkari** [comment]
No worries, let me know if you find anything that is not working :+1:
